iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/IcebergTableUtil.java

```diff
@@ -165,6 +165,41 @@ static Table getTable(Configuration configuration, Properties properties) {
     return getTable(configuration, properties, false);
   }
 
+  static Snapshot getTableSnapshot(Table table, org.apache.hadoop.hive.ql.metadata.Table hmsTable) {
```

Comment:
There is already a method in this class with the same name, same input types, but reverse order:
public static Snapshot getTableSnapshot(org.apache.hadoop.hive.ql.metadata.Table hmsTable, Table table)

It is confusing, and hard to understand what is the difference. Please use the same order of parameters, as the old one, and add other name which expresses the difference.

Reply:
`table` is the main argument and `hmsTable` is supplementary config provider